TabletDriverFilters/TabletFilterSmoothing.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using TabletDriverPlugin;
+using TabletDriverPlugin.Attributes;
+using TabletDriverPlugin.Tablet;
+
+namespace TabletDriverFilters
+{
+    using static Math;
+
+    [PluginName("TabletDriver Smoothing Filter")]
+    public class TabletFilterSmoothing : IFilter
+    {
+        private DateTime? _lastFilterTime;
+        private Point _lastPos;
+        private float _timerInterval;
+        public float _threshold = 0.63f;
```

Comment:
We could define this as `const` since it's never changed, iirc it also improves performance to use constants as well.